### DISCUSSION TITLE
Client-side encryption in Velero

### DISCUSSION THREAD

**Discussion by smarimus** [vmware-tanzu/velero#7736] [Community support Q&A] [answered]
Hi Team,
What are the options available for client-side encryption in Velero?

We are planning to use the s3 as the backup storage location which has multiple SSE options. 
However, there's requirement to clien-side encryption using customer key(CSE-C).

Thanks

**Comment by kaovilai** [comment, marked as answer]
Please follow https://github.com/vmware-tanzu/velero/issues/3218

CSE-C is not yet available, only SSE-C.

**Reply by smarimus** [reply]
Thanks
